.ansible-lint

```diff
@@ -0,0 +1,12 @@
+---
+parseable: true
+skip_list:
+  - ANSIBLE0006
+  - ANSIBLE0012
+  - ANSIBLE0016
+  - '204'  # [E204] Lines should be no longer than 120 chars
+  # E602 https://github.com/ansible/ansible-lint/issues/450
+  - '602'  # [E602] Don't compare to empty string
+  # E405 is too new and we need to enable it in a separate change
```

Comment:
405 was completely removed in the current version of ansible-lint.

Reply:
Well, this CR is old ;)

Reply:
Well, you'll need to rebase/fixup anyways, so I thought you might want to keep this in mind when going through everything again... ;-)

Would you rather see #4411 merged and then fix lints one-by-one in subsequent PRs (maybe also adding this pre-commit tool into the pipeline) or would you rather have this done at once?